kettle/make_json.py

```diff
@@ -103,11 +107,11 @@ def populate_path_to_job_and_number(self):
             if self.path.startswith(bucket):
                 prefix = meta['prefix']
                 break
-
+        else:
```

Comment:
is this supposed to be part of the `if` in the for loop?

Reply:
It actually is not. It is one of those odd `for - else` statements. I never see them and it is probably the reason it got lost in translation.

Reply:
sounds like it tripped us both up, maybe add a comment that this is intentional and/or why?